### PR TITLE
[Docs] Add VBUS hack for teensys

### PR DESCRIPTION
**Pull request by drashna**
## Description

Adds section on how to add VBUS support to Teensy 2.0 and Teensy++ 2.0, so that Split USB Detect isn't needed. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: docs/feature_split_keyboard.md [hunk 1]
```diff
@@ -237,6 +237,26 @@ This sets the maximum timeout when detecting master/slave when using `SPLIT_USB_
...
+
+* A knife (x-acto knife, ideally)
+* A solder station or hot air station
+* An appropriate Schokkty diode, such as the [PMEG2005EH](https://www.digikey.com/en/products/detail/nexperia-usa-inc/PMEG2005EH,115/1589924)
+
+You'll need to cut the small trace between the 5V and center pads on the back of the Teensy.
+
+Once you have done that, you will want to solder the diode from the 5V pad to the center pad.
+
+I had to use the 5V pad from the regulator block above as the pads were too small and placed too closely together.
```
Comment (fauxpark) [inline docs/feature_split_keyboard.md:256]: Not sure about the use of first person in docs :P

Comment (drashna) [reply docs/feature_split_keyboard.md:256]: Ah, yeah, true. 

```suggestion
You may need to use the 5V pad from the regulator block above as the pads were too small and placed too closely together to place the Schottky diode properly. 
```

Should I also mention that I had NO issue with the size on the fake teensy? Only my genuine teensys had this issue?

File: docs/feature_split_keyboard.md [hunk 2]
```diff
@@ -237,6 +237,26 @@ This sets the maximum timeout when detecting master/slave when using `SPLIT_USB_
 ```
 This sets the poll frequency when detecting master/slave when using `SPLIT_USB_DETECT`
 
+## Hardware Considerations and mods
+
+While most any Pro Micro can be used, micro controllers like the AVR teensy's and most (if not all) ARM boards require the Split USB Detect.
```
Comment (fauxpark) [inline docs/feature_split_keyboard.md:242]:
```suggestion
While most any Pro Micro can be used, micro controllers like the AVR Teensys and most (if not all) ARM boards require the Split USB Detect.
```

File: docs/feature_split_keyboard.md [hunk 3]
```diff
@@ -237,6 +237,26 @@ This sets the maximum timeout when detecting master/slave when using `SPLIT_USB_
 ```
 This sets the poll frequency when detecting master/slave when using `SPLIT_USB_DETECT`
 
+## Hardware Considerations and mods
```
Comment (fauxpark) [inline docs/feature_split_keyboard.md:240]:
```suggestion
## Hardware Considerations and Mods
```

File: docs/feature_split_keyboard.md [hunk 4]
```diff
@@ -237,6 +237,26 @@ This sets the maximum timeout when detecting master/slave when using `SPLIT_USB_
...
+* A solder station or hot air station
+* An appropriate Schokkty diode, such as the [PMEG2005EH](https://www.digikey.com/en/products/detail/nexperia-usa-inc/PMEG2005EH,115/1589924)
+
+You'll need to cut the small trace between the 5V and center pads on the back of the Teensy.
+
+Once you have done that, you will want to solder the diode from the 5V pad to the center pad.
+
+You may need to use the 5V pad from the regulator block above as the pads were too small and placed too closely together to place the Schottky diode properly.
+
+![Teensy++ 2.0](https://cdn.discordapp.com/attachments/440911374571274240/779382993289871360/20201120_082838.jpg)
```
Comment (fauxpark) [inline docs/feature_split_keyboard.md:258]:
```suggestion
![Teensy++ 2.0](https://i.imgur.com/BPEC5n5.png)
```

**Review by fauxpark — CHANGES_REQUESTED** [review]
Some slight tweaks, LGTM otherwise.

**Review by fauxpark — APPROVED** [review]

File: docs/feature_split_keyboard.md [hunk 5]
```diff
@@ -237,6 +237,26 @@ This sets the maximum timeout when detecting master/slave when using `SPLIT_USB_
...
+
+While most any Pro Micro can be used, micro controllers like the AVR Teensys and most (if not all) ARM boards require the Split USB Detect.
+
+However, with the Teensy 2.0 and Teensy++ 2.0, there is a simple hardware mod that you can perform to add VBUS detection, so you don't need the Split USB detection option.
+
+You'll only need a few things:
+
+* A knife (x-acto knife, ideally)
+* A solder station or hot air station
+* An appropriate Schokkty diode, such as the [PMEG2005EH](https://www.digikey.com/en/products/detail/nexperia-usa-inc/PMEG2005EH,115/1589924)
```
Comment (noroadsleft) [inline docs/feature_split_keyboard.md:250]:
```suggestion
* An appropriate Schottky diode, such as the [PMEG2005EH](https://www.digikey.com/en/products/detail/nexperia-usa-inc/PMEG2005EH,115/1589924)
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**